.github/ISSUE_TEMPLATE/docs_request.yml

```
@@ -0,0 +1,24 @@
+name: 'Docs Request for an Update or Improvement'
```

For consistency with other issue templates, it would be good to change like below:
```suggestion
name: 📚 Documentation
```